proto/celestia/core/v1/blob/blob.proto

```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
-package celestia.core.v1.blob;
+package square.blob.v1;
```

Comment:
[question] Since this package name was modified, does the filename also need to be updated to remove `celestia/core`?

Reply:
I was planning on modify the directory structure in a follow up PR

Reply:
Works for me